### ISSUE TITLE
Notification icon does not show up on Binder or JupyterHub

### ISSUE THREAD

**Issue by krassowski** [mwakaba2/jupyterlab-notifications#27]
The use of absolute path should be prefixed with the base to work correctly otherwise binder and hub will give 404 errors:

> /hub/static/favicon.ico:1 Failed to load resource: the server responded with a status of 404 ()

I think that getting the base URL from `PageConfig` should be sufficient:

```typescript
import { PageConfig } from '@jupyterlab/coreutils';

const base = PageConfig.getBaseUrl();
const icon = base + 'static/favicon.ico'
```

I have not tested it though.

**Comment by mwakaba2** [comment]
New fix is released in v0.3.2

**Comment by krassowski** [comment]
It works great, thank you!
